node-fs-writefile/cp.js

```diff
@@ -0,0 +1,22 @@
+import { writeFile } from 'node:fs';
+import { readFile } from 'node:fs/promises';
+
+try {
+  const data = await readFile(String(process.argv[2]), 'utf8');
```

Comment:
It would be best to do the readFile without an encoding. With it, you can only copy text files. If you leave it off you'll be able to copy images, etc., as well.